Framework/DataHandling/test/LoadILLDiffractionTest.h

```diff
@@ -478,6 +486,13 @@ class LoadILLDiffractionTest : public CxxTest::TestSuite {
     TS_ASSERT(run.hasProperty("ScanType"));
     const auto type = run.getLogData("ScanType");
     TS_ASSERT_EQUALS(type->value(), "DetectorScan");
+    checkTimeFormat(outputWS);
+  }
+
+  void checkTimeFormat(MatrixWorkspace_const_sptr outputWS) {
```

Comment:
This is repeated in all the loaders, we should rather put this somewhere in test helpers.

Reply:
Do you have an idea in which one it would be most fitting ? I don't find any that matches

Reply:
Indeed, there doesn't seem to be a good place for that, I think it's alright then.